metarecord/views/function.py

```diff
@@ -84,8 +85,9 @@ def _create_new_version(self, function_data, copy_from_previous=False, phases=No
         user_data = {'created_by': user, 'modified_by': user}
 
         phase_data = function_data.pop('phases', [])
+        if function_data.get("bulk_update"):
+            function_data["bulk_update"] = BulkUpdate.objects.get(id=function_data["bulk_update"])
 
-        function_data.update(user_data)
```

Comment:
Hmm, is this intentional?

Reply:
no, it's not. Wonder how the tests still worked.